src/main/java/com/tabnine/userSettings/AppSettingsComponent.kt

```diff
@@ -81,15 +89,28 @@ class AppSettingsComponent {
             .addLabeledComponent("Log File Path (requires restart): ", logFilePathComponent, 1, false)
             .addLabeledComponent("Log level (requires restart): ", logLevelComponent, 1, false)
 
+        if (Config.CHANNEL == "onprem") {
```

Comment:
any way to restart without the user interaction?

Reply:
i'll check

Reply:
i think it is possible, but not worth our while